server/form-pages/apply/area-and-funding/funding-information/alternativeID.ts

```diff
@@ -0,0 +1,137 @@
+import type { TaskListErrors } from '@approved-premises/ui'
+import { Cas2Application as Application } from '@approved-premises/api'
+import { Page } from '../../../utils/decorators'
+import TaskListPage from '../../../taskListPage'
+import { convertKeyValuePairToCheckboxItems } from '../../../../utils/formUtils'
+import errorLookups from '../../../../i18n/en/errors.json'
+import { nameOrPlaceholderCopy } from '../../../../utils/utils'
+import { getQuestions } from '../../../utils/questions'
+
+const applicationQuestions = getQuestions('')['funding-information']['alternative-identification']
+
+const alternativeIDOptions = applicationQuestions.alternativeIDDocuments.answers
```

Comment:
is there a reason why sometimes there are variables declared outside of the class? I don't have a strong opinion either way, but just curious as elsewhere I've just been assigning things like `questions` when they are declared in the class https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/blob/314d499ae6d0c830fb46afd28e3b5aeb23f8764b/server/form-pages/apply/offence-and-licence-information/offending-history/anyPreviousConvictions.ts

Reply:
I think I've been doing it in cases where the body type definition relies on the answer keys (as on line 15 here). I don't think we can define types within the class...?